api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -547,8 +547,8 @@ message HttpConnectionManager {
   // on stream close, when the HTTP request is complete. If this field is set, the HCM will flush access
   // logs periodically at the specified interval. This is especially useful in the case of long-lived
   // requests, such as CONNECT and Websockets. Final access logs can be detected via the
-  // `requestComplete()` method of `StreamInfo` in access log filters, or thru the `%DURATION%` substitution
-  // string.
+  // `requestComplete()` method of `StreamInfo` in access log filters, or thru the `%STREAM_STATE%`
```

Comment:
That's not a breaking change. This line should have changed in this PR (but was forgotten): https://github.com/envoyproxy/envoy/pull/26256
When this PR will be merged, then it will be indeed a breaking change, so I'd rather change it now: https://github.com/envoyproxy/envoy/pull/26203